package.json

```diff
@@ -49,128 +49,83 @@
-    "clsx": "^1.2.1"
+    "@storybook/components": "^7.1.1",
+    "classnames": "^2.3.2"
   },
   "devDependencies": {
     "@babel/core": "^7.22.8",
-    "@babel/eslint-parser": "^7.22.7",
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-syntax-flow": "^7.22.5",
-    "@babel/plugin-transform-react-jsx": "^7.22.5",
     "@babel/plugin-transform-runtime": "^7.22.7",
     "@babel/preset-env": "^7.22.7",
     "@babel/preset-react": "^7.22.5",
     "@babel/preset-typescript": "^7.22.5",
     "@babel/runtime": "^7.22.6",
-    "@popperjs/core": "^2.11.8",
     "@rollup/plugin-babel": "^6.0.3",
     "@rollup/plugin-commonjs": "^25.0.2",
     "@rollup/plugin-node-resolve": "^15.1.0",
     "@rollup/plugin-terser": "^0.4.3",
-    "@storybook/addon-actions": "^7.0.26",
-    "@storybook/addon-docs": "^7.0.26",
-    "@storybook/addon-essentials": "^7.0.26",
-    "@storybook/addon-interactions": "^7.0.26",
-    "@storybook/addon-links": "^7.0.26",
+    "@storybook/addon-actions": "^7.1.1",
+    "@storybook/addon-essentials": "^7.1.1",
+    "@storybook/addon-interactions": "^7.1.1",
+    "@storybook/addon-links": "^7.1.1",
     "@storybook/addon-styling": "^1.3.4",
-    "@storybook/addons": "^7.0.26",
-    "@storybook/blocks": "^7.0.26",
-    "@storybook/manager-api": "^7.1.0",
-    "@storybook/node-logger": "^7.0.26",
-    "@storybook/react": "^7.0.26",
-    "@storybook/react-webpack5": "^7.0.26",
+    "@storybook/addons": "^7.1.1",
+    "@storybook/blocks": "^7.1.1",
+    "@storybook/manager-api": "^7.1.1",
+    "@storybook/react": "^7.1.1",
+    "@storybook/react-webpack5": "^7.1.1",
     "@storybook/testing-library": "^0.2.0",
-    "@storybook/theming": "^7.1.0",
-    "@svgr/rollup": "^8.0.1",
-    "@svgr/webpack": "^8.0.1",
+    "@storybook/theming": "^7.1.1",
     "@testing-library/dom": "^9.3.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^14.0.0",
-    "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^14.4.3",
-    "@types/bluebird": "^3.5.38",
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^29.5.3",
     "@types/node": "^20.4.1",
     "@types/react": "^18.2.14",
     "@types/react-dom": "^18.2.6",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
-    "babel-core": "^6.26.3",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
+    "@typescript-eslint/parser": "^6.2.0",
     "babel-jest": "^29.3.1",
-    "babel-loader": "^8.2.5",
-    "babel-plugin-react-docgen": "^4.2.0",
-    "babel-plugin-react-require": "^3.1.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
+    "babel-plugin-react-require": "^4.0.1",
     "boxt": "^1.1.1",
     "camelcase": "^7.0.1",
-    "chalk": "^4.1.2",
+    "chalk": "^5.3.0",
```

Comment:
Where are we using chalk?

Reply:
Both chalk and execa are used in release and prerelease scripts, which I copied from monday-ui-react-core